standards/SPIFFE_Federation.md

```diff
@@ -0,0 +1,316 @@
+
+After establishing a TLS connection to the bundle endpoint and authenticating the presented server certificate, clients issue an HTTP GET for the path specified by the endpoint URL. The body of the response is a SPIFFE bundle. Clients MUST know the name of the trust domain that the endpoint URL represents prior to retrieving the trust bundle, ideally via explicit configuration; please see the [Security Considerations](#7-security-considerations) section for more information.
+
+Bundle endpoint servers MAY respond with HTTP redirect (as defined by [RFC 7231 section 6.4][8]).  Bundle endpoint clients SHOULD follow the redirect if the URL meets all the requirements of a valid bundle endpoint URL.  When connecting to the new URL, the same TLS considerations MUST be applied as connecting to the original URL.  Bundle endpoint clients SHOULD use the configured endpoint URL for each bundle refresh and SHOULD NOT permanently store the location for future fetches. See the [Security Considerations](#7-security-considerations) for more information. 
+
+#### 5.2.2. SPIFFE Authentication (`https_spiffe`)
+The `https_spiffe` profile uses an X509-SVID issued by a SPIFFE trust domain (as opposed to a certificate issued by public certificate authorities). This profile allows bundle endpoints to avoid the use of network locators as a form of server identity and additionally supports automated root CA rotation and revocation via standard SPIFFE mechanisms.
+
+In addition to the endpoint parameters required of all profiles, the `https_spiffe` profile requires additional endpoint client parameters as described in Endpoint Parameters below.
+
+##### 5.2.2.1. Endpoint URL Requirements
+Bundle endpoint URLs utilizing `https_spiffe` MUST have the scheme set to `https` and MUST NOT include userinfo in the authority component.  Other components of the URL (as defined by [RFC 3986 section 3][7]) are not constrained by this specification. 
+
+For example, the URL `https://host.example.com/trust_domain` is a valid SPIFFE bundle endpoint URL for the `https_spiffe` profile type.
+
+##### 5.2.2.2. Endpoint Parameters
+Bundle endpoint clients using the `https_spiffe` profile MUST be configured with the SPIFFE ID of the bundle endpoint server as well as a secure method for obtaining the trust bundle of the endpoint server's trust domain. A self-serving bundle endpoint is one in which the bundle endpoint server’s SPIFFE ID resides in the same trust domain as the bundle being fetched. Configured bundle endpoints may or may not be self-serving.
```

Comment:
Can we bold "self-serving bundle endpoint" to help readers identify this as a definition of a new term? Or put it in quotes or... something...

Reply:
```suggestion
Bundle endpoint clients using the `https_spiffe` profile MUST be configured with the SPIFFE ID of the bundle endpoint server as well as a secure method for obtaining the trust bundle of the endpoint server's trust domain. A **self-serving** bundle endpoint is one in which the bundle endpoint server’s SPIFFE ID resides in the same trust domain as the bundle being fetched. Configured bundle endpoints may or may not be self-serving.
```